sdk/tests/tutorials/instruments/test_complex_instruments.py

```diff
@@ -106,3 +106,70 @@ def test_create_fx_option(self):
         self.assertEqual(
             saved_fx_option.is_delivery_not_cash, fx_option.is_delivery_not_cash
         )
+
+    def test_create_term_deposit(self):
+
+        term_deposit = models.TermDeposit(
+            start_date=datetime(2020, 2, 5, 00, tzinfo=pytz.utc),
+            maturity_date=datetime(2020, 2, 8, 00, tzinfo=pytz.utc),
+            contract_size=1000000,
+            flow_convention=models.FlowConventions(
+                scope=None,
+                code=None,
+                currency="GBP",
+                payment_frequency="6M",
+                roll_convention="MF",
+                day_count_convention="Act365",
+                payment_calendars=[],
+                reset_calendars=[],
+                settle_days=1,
+                reset_days=0
+                ),
+            rate=0.03,
+            instrument_type="TermDeposit"
+        )
+        upsert_term_deposit = self.instruments_api.upsert_instruments(request_body={
+        "upsert_request_001": models.InstrumentDefinition(
+            name="Term Deposit Test3%",
+            identifiers={"ClientInternal": models.InstrumentIdValue("test_td")},
+            definition=term_deposit
+        )
+    })
+        # Assert instrument was created
+        self.assertIsNotNone(upsert_term_deposit.values["upsert_request_001"].instrument_definition)
+        self.assertIsNotNone(upsert_term_deposit.values["upsert_request_001"].lusid_instrument_id)
+
+    def test_create_zero_coupon_bond(self):
+
+        zero_coupon_bond = models.Bond(
+            start_date=datetime(2020, 2, 7, 00, tzinfo=pytz.utc),
+            maturity_date=datetime(2020, 9, 18, 00, tzinfo=pytz.utc),
+            dom_ccy="GBP",
+            principal=100000,
+            coupon_rate=0,
+            flow_conventions=models.FlowConventions(
+                scope=None,
+                code=None,
+                currency="GBP",
+                payment_frequency="0Invalid",
+                roll_convention="None",
+                day_count_convention="Invalid",
+                payment_calendars=[],
+                reset_calendars=[],
+                settle_days=2,
+                reset_days=2
+                ),
+            identifiers={},
+            instrument_type="Bond"
+        )
+
+        upsert_zero_coupon_bond = self.instruments_api.upsert_instruments(request_body={
+            "upsert_request_001": models.InstrumentDefinition(
+                name="Zero Coupon Test",
+                identifiers={"ClientInternal": models.InstrumentIdValue("test_zc")},
```

Comment:
Think "ZCB" is the more common abbreviation?

Reply:
As above for the "test" identifiers: `test_zc` -> `ZeroCouponBondInstrument` and `name = "Zero Coupon Bond Example"